src/main/java/com/onepagecrm/models/serializers/EmailMessageSerializer.java

```diff
@@ -0,0 +1,110 @@
+package com.onepagecrm.models.serializers;
+
+import com.onepagecrm.exceptions.OnePageException;
+import com.onepagecrm.models.Email;
+import com.onepagecrm.models.EmailMessage;
+import com.onepagecrm.models.EmailRecipients;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Logger;
+
+public class EmailMessageSerializer extends BaseSerializer {
+
+    private static final Logger LOG = Logger.getLogger(EmailMessageSerializer.class.getName());
+
+    public static List<EmailMessage> fromString(String responseBody) throws OnePageException {
+        List<EmailMessage> result = new ArrayList<>();
+        try {
+            String parsedResponse = (String) BaseSerializer.fromString(responseBody);
+            JSONObject responseObject = new JSONObject(parsedResponse);
+            return fromJsonArray(responseObject.getJSONArray("email_messages"));
```

Comment:
@fraggjkee - can we add constants for keys, for example - here it would be:
```
BaseSerializer.java
---
    // EMAIL TAGS
    public static final String EMAIL_MESSAGES_TAG = "email_messages";
    public static final String SEND_TIME_TAG = "send_time";
    // ... etc.
```

Reply:
Done